### PR TITLE
Displaying recipes

### LINKED ISSUE #8: Create API to add new or editing data to JSON recipes
(CRUD - create read update delete)

### PR DESCRIPTION
**Pull request by EwaTrzemzalska**
Resolves #8

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -20,47 +20,29 @@
...
+  <form class="uk-search uk-search-default margin-medium">
+    <div class="uk-margin">
+      <label for="searching-field" class="font-size-medium">WHAT ARE YOU LOOKING FOR?</label>
+      <input id="searching-field" class="uk-search-input" type="search" placeholder="Search...">
+      <button type="submit" class="uk-search-icon-flip" uk-search-icon></button>
+
+    </div>
   </form>
 
+  <!-- zrob ula i wyswietl najprostsze lisy z tytulami -->
```
Comment (smogg) [inline index.html]: What's that?

File: js/main.js [hunk 2]
```diff
@@ -1 +1,31 @@
...
+const {
+  getRecipes,
+  getRecipe,
+  createRecipe,
+  updateRecipe,
+  deleteRecipe
+} = require('../js/recipe_manager')
+
+// zrobic loopa ktory consolloguje recipesy
+// stworz funkcje ktorej jak przeslesz recipe to doda ja do listy na stronie
```
Comment (smogg) [inline js/main.js]: Oops

File: js/main.js [hunk 3]
```diff
@@ -1 +1,31 @@
 console.log('Hello World')
```
Comment (smogg) [inline js/main.js]: Is that console log helpful in any way?

File: js/main.js (hunk 2 above)
Comment (EwaTrzemzalska) [reply js/main.js]: oooooops...

**Review by smogg — APPROVED** [review]

**Pull request merged**